common-compose.yml

```diff
@@ -36,6 +36,7 @@ services:
       - PATH_PREFIX=/api/v1
       - CROMWELL_CREDENTIALS=/app/jobs/config.json
       - CROMWELL_URL=${CROMWELL_URL:-https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1}
+      - USE_CAAS=0
```

Comment:
nit: Since these environment vars now differ between cromwell compose files, can you move them out of the common compose and only into the extended compose files? Our general pattern has been to only add environment vars which are not overriden in extended compose files.

Reply:
tiny nit: I may also just call this `REQUIRES_AUTH` to match the dsub implementation. It looks like you only switch on this flag for returning the capabilities response and getting the bearer token from the header, which isn't really specific to CAAS. 

We may be able to extract some of the auth token stuff out to jm_utils in the future.

Reply:
I kept this specific to using caas for the future case where we want users to log in with a gmail account to the job manager so they can see their preferences, but still connect to a cromwell instance that requires just a username/password. In that case, `REQUIRES_AUTH` would be true but we would not send a bearer token to cromwell.

Reply:
Gotcha, makes sense!